examples/src/main/java/io/opencensus/examples/http/jetty/client/HelloWorldClient.java

```diff
@@ -68,6 +68,10 @@ public static void main(String[] args) throws Exception {
     initTracing();
     initStatsExporter();
 
+    // For HTTPS URLs, uncomment the lines below using the SslContextFactory
+    // HttpClientTransport transport = new HttpClientTransportOverHTTP();
+    // SslContextFactory sslCf = new SslContextFactory();
+    // OcJettyHttpClient httpClient = new OcJettyHttpClient(transport, sslCf, null, null);
```

Comment:
Please remove this for now. I'll update with my PR after 0.20.0 is released.

Reply:
OK. Just removed it with a new commit.